DESCRIPTION

```diff
@@ -0,0 +1,47 @@
+Package: spectratrait
```

Comment:
@JulienLamour @neo0351 what are your thoughts about this as a package name for a packaged version of this?

Reply:
spectratrait works for me.  could also maybe shorten it to spectrait.